manifests/server.pp

```diff
@@ -8,11 +8,15 @@
 ) inherits ssh::params {
   include ssh
 
-  if $::kernel == 'Linux' and ! defined(Package[$server_package]) {
-    package { $server_package:
-      ensure => latest,
-      notify => Service['sshd'],
+  if $facts['kernel'] == 'Linux' {
+    if ! defined(Package[$ssh::params::server_package]) {
```

Comment:
@danielparks if we are going to keep doing this lets change this to use `ensure_packages` from stdlib.

Reply:
Oh, thanks for pointing that out. I thought `ensure_packages` broke when the parameters are different, but experimentation confirms that it doesn't.